samples/sampler/stories/default/About.js

```diff
@@ -64,7 +65,17 @@ const HintDialog = kind({
 });
 
 export default {
-	title: 'About/A Tour of Sampler'
+	title: 'About/A Tour of Sampler',
+	parameters: {
+		docs: {
```

Comment:
Oh... you don't need to do this for every story. You could just do it from the preview.js.

Reply:
Oh, I don't know why I didn't think of that. I will fix this. Thank you, Mikyung!

Reply:
I kept the imports sorting if that is ok.